hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/node/DeadNodeHandler.java

```diff
@@ -122,8 +122,8 @@ private void destroyPipelines(final DatanodeDetails datanodeDetails) {
         .ifPresent(pipelines ->
             pipelines.forEach(id -> {
               try {
-                pipelineManager.closePipeline(
-                    pipelineManager.getPipeline(id), false);
+                pipelineManager.closePipeline(id);
```

Comment:
Should we switch the order [here?](https://github.com/apache/ozone/blob/8a11e0b6a9d66dca0e34df87656556ce0b23ab6e/hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/node/DeadNodeHandler.java#L83) 
```
      LOG.info("A dead datanode is detected. {}", datanodeDetails);
      destroyPipelines(datanodeDetails);
      closeContainers(datanodeDetails, publisher);
```

Reply:
Sure.